inspirehep/modules/search/config.py

```diff
@@ -216,3 +216,24 @@
     'year': 'year',
     'control_number': 'recid',
 }
+
+# Specify special boosting when no keyword is given in a query.
+DEFAULT_FIELDS_BOOSTING = {
```

Comment:
Shall we prefix this variable with `SEARCH`? Like `SEARCH_DEFAULT_FIELDS_BOOSTING `